api/interservice/authz/v2/project.proto

```diff
@@ -111,6 +111,8 @@ message ProjectRule {
     string name = 3;
     ProjectRuleTypes type = 4;
     repeated Condition conditions = 5;
+    bool deleted = 6;
```

Comment:
ℹ️Surfacing the deleted and staged/applied status of a rule in the interservice API. Nothing done for `automate-gateway` yet.